security/pkg/pki/ca/ca.go

```diff
@@ -162,7 +163,7 @@ func NewSelfSignedIstioCAOptions(ctx context.Context, caCertTTL, certTTL, maxCer
 
 		options := util.CertOptions{
 			TTL:          caCertTTL,
-			Org:          org,
+			Org:          fmt.Sprintf("%s (%s)", org, uuid.New().String()),
```

Comment:
Consider adding a comment of why using a uuid as the Org (e.g., the benefits it brings).